### PR TITLE
Translated the AdafruitStepperMotor.py and StepperTest.py

### PR DESCRIPTION
**Pull request by jdestremps**
Translated the AdafruitStepperMotor.py and StepperTest.py into both Kotlin and Java classes respectively, and have updated MainActivity to run one test or the other (DC or stepper).  Tested with two stepper motors attached.

### CODE REVIEW

File: AdafruitMotorHat/lib/src/main/java/com/zugaldia/adafruit/motorhat/library/AdafruitStepperMotor.java [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+    }
+}
+
+
+
+
+
+
+/*
+#!/usr/bin/python
```
Comment (zugaldia) [inline AdafruitMotorHat/lib/src/main/java/com/zugaldia/adafruit/motorhat/library/AdafruitStepperMotor.java]: This is looking great - thanks for the contribution. Could you remove this Python code from here before we merge?

**Review by zugaldia — CHANGES_REQUESTED** [review]

**Comment by jdestremps** [comment]
Sure

File: AdafruitMotorHat/lib/src/main/java/com/zugaldia/adafruit/motorhat/library/AdafruitStepperMotor.java (hunk 1 above)
Comment (jdestremps) [reply AdafruitMotorHat/lib/src/main/java/com/zugaldia/adafruit/motorhat/library/AdafruitStepperMotor.java]: Ok, done.

**Review by zugaldia — APPROVED** [review]

**Comment by zugaldia** [comment]
Thank you, @jdestremps, again for the contribution!

**Pull request merged**